doc/dev/release.md

```diff
@@ -327,9 +350,7 @@ The release process for the samples repo is simple:
     $ git push --tags
     ```
 
-### (Post-release) Updating the release notes
-
-Add the following line to the top of the GitHub release notes for `v1.3.0`:
+Then, add the following line to the top of the GitHub release notes for `v1.3.0`:
```

Comment:
I didn't realize that we already have these post-release sections in the doc (I clearly didn't play close enough attention to this during the `v0.12.0` release :grimacing:)

But I think the existing sections capture the intent well enough already. The only changes that are necessary AFAICT, is that we should say that the actual post release steps by the releaser should be to create the tickets to do these, not actually do them.

WDYT?

Reply:
I updated it and keep it simple. Let's me know if you are OK with now.